cincinnati/src/plugins/internal/edge_add_remove.rs

```diff
@@ -8,12 +8,16 @@ use crate::ReleaseId;
 use failure::Fallible;
 
 static DEFAULT_KEY_FILTER: &str = "io.openshift.upgrades.graph";
+pub static DEFAULT_REMOVE_ALL_EDGES_VALUE: &str = "*";
 
 #[derive(Clone, Debug, Deserialize, SmartDefault)]
 #[serde(default)]
 pub struct EdgeAddRemovePlugin {
     #[default(DEFAULT_KEY_FILTER.to_string())]
     pub key_prefix: String,
+
+    #[default(DEFAULT_REMOVE_ALL_EDGES_VALUE.to_string())]
+    pub remove_all_edges_value: String,
```

Comment:
Does it make sense to have this configurable? It needs to be a special reserved identifier anyway, otherwise we cannot disambiguate it from a normal release version label.

Reply:
It makes sense to me to have this configurable as a way to expose it, since we currently don't have a proper way of communicating these decisions. Since we almost have the configuration infrastructure in place this poses no overhead.

Reply:
Ack, let's keep it then. I'm mostly concerned with deployment configuring this in a silly (i.e. ambiguous way).